src/herder/HerderImpl.cpp

```diff
@@ -762,6 +765,25 @@ HerderImpl::triggerNextLedger(uint32_t ledgerSeqToTrigger)
         }
     }
 
+    if (!upgrades.empty())
+    {
+        auto message =
+            fmt::format("Proposing herder configuration upgrades: {0}; network "
```

Comment:
it would be cleaner to either use the same names than `Upgrades::toString` (for example `BASE_FEE` instead of `baseFee`) or better yet, use `Upgrades::toString` on the network values.